Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -676,16 +670,7 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
             onSubmit={handleCreateDialogSubmit}
           />
         )}
-        {showAddSkillDialogModal && (
-          <CreateSkillModal
```

Comment:
Can we pass the projectId along with the CreateSkillModal. The reason being when we support multiple bots we are calling the dispatcher function with the project ID rather than fetching the current project ID asynchronously in the dispatcher.

Reply:
https://github.com/microsoft/BotFramework-Composer/pull/4117/files#diff-566cb1ffd02936030db44dde3e418609L20